advocacy_docs/supported-open-source/pgbackrest/06-use_case_1.mdx

```diff
@@ -0,0 +1,212 @@
+repo1-path=/repo1
+repo1-azure-container=container-name
+repo1-azure-account=pgbackrest
+repo1-azure-key=***
+```
+
+To use the shared access signatures, set the `repo1-azure-key-type` option to **sas** and the `repo1-azure-key` option to the shared access signature token.
+
+##### Recommended Settings
+
+Use those settings to enable encryption, parallel operations and ensure displaying enough information in the console and in the log file:
+
+```ini
+repo1-cipher-type=aes-256-cbc
+repo1-cipher-pass=FIXME
+process-max=FIXME
+log-level-console=info
+log-level-file=debug
+start-fast=y
+delta=y
+```
+
+Adjust the encryption passphrase and the maximum number of processes to use for compression usage and file transfer.
+
+See the [Recommended settings](04-recommended_settings) page for more details.
+
+##### Retention Policy
+
+To be able to delete old backups and archives, you have to define a retention policy. Use the following setting to only keep the latest full backup:
+
+```ini
+repo1-retention-full=1
+```
+
+See the [Retention policy](05-retention_policy) page for more options and details.
+
+#### Stanza Section
+
+The `[demo]` stanza section will be different on each database server and should contain:
+
+* the location of the **PostgreSQL** configuration files
+
+```ini
+pg1-path=/var/lib/pgsql/13/data
+```
+
+or for **EDB Postgres Advanced Server**
+
+```ini
+pg1-path=/var/lib/edb/as13/data
+pg1-user=enterprisedb
+pg1-port=5444
+```
+
+* for every other **PostgreSQL** node
+
+```ini
+pg2-host=other_node_hostname_or_ip
+pg2-path=/var/lib/pgsql/13/data
+```
+
+or for **EDB Postgres Advanced Server**
+
+```ini
+pg2-host=other_node_hostname_or_ip
+pg2-host-user=enterprisedb
+pg2-path=/var/lib/edb/as13/data
+```
+
+* specific recovery option to set the replication `primary_conninfo` after the restore command is executed:
+
+```ini
+recovery-option=primary_conninfo=host=primary_vip user=replication_user ...
+```
+
+#### Setup Archiving
+
+Once _pgBackRest_ is configured, set up the database archiver process on each node:
+
+```ini
+# postgresql.conf
+archive_mode = on
+archive_command = 'pgbackrest --stanza=demo archive-push %p'
+```
+
+As changing the `archive_mode` requires a service restart and changing the `archive_command` requires only a configuration reload, we recommend enabling `archive_mode` with an empty `archive_command` (or pointing to `/bin/true`) when initiating a new database cluster.
+
+See the [Quick start](03-quick_start) page to set up a simple **demo** stanza configuration.
+
+Assume the identity of the system user (`postgres` or `enterprisedb`) and initiate the _pgBackRest_ repository from the primary server:
+
+```bash
+$ pgbackrest --stanza=demo stanza-create
+```
+
+Check the configuration and the archiving process:
+
+```bash
+$ pgbackrest --stanza=demo check
+```
+
+### Backup and Restore
+
+See the **Quick start** [backups](03-quick_start#backups) sections for more details about those two commands.
```

Comment:
It seems it didn't pulled the latest sources with both links to backups and restore...

```
See the **Quick start** [backups](03-quick_start#backups) and [restore](03-quick_start#restore) sections for more details about those two commands.
```

Reply:
That's... peculiar